pkg/storage/cacher_test.go

```diff
@@ -335,7 +338,6 @@ func TestFiltering(t *testing.T) {
 	}
 	defer watcher.Stop()
 
-	verifyWatchEvent(t, watcher, watch.Added, podFoo)
```

Comment:
>
I actually want to test the normal scenario, passing in a resourceVersion and making sure we don't get that event.

OK - that makes sense. So how about leaving one of the two testcases as it was (to test skipping the event), and changing the other one to check if it will be delivered?

>
 If we passed resourceVersion-1, we wouldn't know if the cache was skipping or just returning all events it had

I don't understand the part "or just returning all events it had".